src/libcore/num/flt2dec/strategy/grisu.rs

```diff
@@ -430,7 +481,7 @@ pub fn format_shortest_opt(d: &Decoded,
 /// The shortest mode implementation for Grisu with Dragon fallback.
 ///
 /// This should be used for most cases.
-pub fn format_shortest(d: &Decoded, buf: &mut [u8]) -> (/*#digits*/ usize, /*exp*/ i16) {
+pub fn format_shortest(d: &Decoded, buf: &mut [u8]) -> (usize, i16) {
```

Comment:
Lost comments here, they should be moved outside the decl, I guess (although a very recent change to rustfmt might have fixed this).